keys/src/hpke.rs

```diff
@@ -0,0 +1,156 @@
+use borsh::{self, BorshDeserialize, BorshSerialize};
```

Comment:
FYI I want Michel to take a look at this to make sure we are not misusing anything here, will let you know how it goes

Reply:
sounds good

Reply:
@itegulov  Are we good to go?